mindsdb/integrations/handlers/langchain_handler/skill_tool.py

```diff
@@ -41,57 +48,66 @@ def _make_text_to_sql_tools(skill: db.Skills, llm, executor) -> List:
     return sql_database_tools
 
 
+def _make_text_to_sql_tools(skill: db.Skills, llm, executor) -> List:
+    '''
+    alternative to _make_text_to_sql_langchain_tools
+    creates custom tool instead of using SQLDatabaseToolkit
```

Comment:
Perhaps document _why_ this would be preferred over `_make_text_to_sql_langchain_tools`.

Reply:
the idea is: 
replace 4 tools generated by langchain (sql_db_query, sql_db_schema, sql_db_list_tables, sql_db_query_checker)
to one tool (sql_db_query) and, as we already know list of tables, provide all information about tables to use.

Reply:
Perfect. Then, let's add this to the docstring:

```
we replace all 4 tools with a single one (`sql_db_query`) and, as we already know the list of tables and their relevant information, we provide it directly instead.
```